.github/workflows/bump-auxiliary-packages.yml

```diff
@@ -0,0 +1,56 @@
+name: Bump Auxiliary Packages
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  update_generated_files:
+    name: Bump packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Github App Token
+        uses: mongodb-js/devtools-shared/actions/setup-bot-token@main
+        id: app-token
+        with:
+          app-id: ${{ vars.DEVTOOLS_BOT_APP_ID }}
+          private-key: ${{ secrets.DEVTOOLS_BOT_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
+        with:
+          # don't checkout a detatched HEAD
+          ref: ${{ github.head_ref }}
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20.16.0
+          cache: "npm"
+
+      - name: Install npm@10.2.4
+        run: |
+          npm install -g npm@10.2.4
+
+      - name: Install Dependencies
+        run: |
+          npm -v
+          npm ci
+
+      - name: Bump packages
+        env:
+          LAST_BUMP_COMMIT_MESSAGE: "chore(release): bump auxiliary packages"
```

Comment:
[Extreme nit] Why are we passing in the commit message as env variable instead of just hardcoding it in the script?

Reply:
yeah I did find it a little weird, inherited from compass script, will remove